tests/cli/bootstrap.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
I'm not totally against calling this "bootstrap" (although there might be confusion with the `bootstrap.php` of `tests/`. My alternative suggestion would be to move `bootstrap.php` one level up, call it generate_inspectors and make it an executable without php suffix. Then you can also rename the cli namespace.